integration_tests/scripts/run_demos.py

```diff
@@ -25,8 +25,7 @@ def run_demo(demo: str, format: str):
     demo_dir = os.path.join(project_dir, demo)
     print("Running demo: {}".format(demo))
 
-    subprocess.run(["docker", "compose", "up", "-d", "-e",
-    "ENABLE_TELEMETRY=false"], cwd=demo_dir, check=True)
+    subprocess.run(["docker", "compose", "up", "-d"], cwd=demo_dir, check=True)
```

Comment:
Please ignore this. This is a temporary fix and @odysa will draw another PR to fix it properly.